src/main/java/StartServer.java

```diff
@@ -0,0 +1,31 @@
+import server.Server;
+
+import java.io.IOException;
+import java.net.Socket;
+import java.net.SocketException;
+
+import static Validator.Validator.*;
+
+public class StartServer {
```

Comment:
I love this name.  Have you looked into the UML / modeling video I linked a while ago?  It could be time to talk about a few patterns (namely the service pattern).